definitions/tools/add_strelka_gt.cwl

```diff
@@ -55,6 +55,27 @@ requirements:
                             $id++;
                         }
 
+                        # parse GT, handling unparseable case with stderr
+                        my $parsed_n_gt = parse_gt($n_gt_str, \%ids);
+                        if ($n_gt_info eq "ref") {
+                            $n_gt = "0/0";
+                        } elsif (defined($parsed_n_gt)) {
+                            $n_gt = $parsed_n_gt;
+                        } else {
+                            my $id_keys = join(" ", sort keys(%ids));
+                            print STDERR "parse_gt for n_gt=\"$n_gt_str\" and ids=\"$id_keys\" resulted in undefined\n";
+                            next;
+                        }
+
+                        my $parsed_t_gt = parse_gt($t_gt_str, \%ids);
+                        if (defined($parsed_t_gt)) {
+                            $t_gt = $parsed_t_gt;
+                        } else {
+                            my $id_keys = join(" ", sort keys(%ids));
+                            print STDERR "parse_gt for t_gt=\"$t_gt_str\" and ids=\"$id_keys\" resulted in undefined\n";
+                            next;
+                        }
+
                         $n_gt = $n_gt_info eq 'ref' ? '0/0' : parse_gt($n_gt_str, \%ids);
```

Comment:
Looks like if we get this far we're throwing away all the work in the new code and going back to the old way.

Reply:
agh I forgot to delete the old lines when I copied the new ones over from the WDL I tested in. Fixing now